sdk/formrecognizer/azure-ai-formrecognizer/samples/async_samples/sample_get_manual_validation_info_async.py

```diff
@@ -0,0 +1,93 @@
+    key = os.environ["AZURE_FORM_RECOGNIZER_KEY"]
+    model_id = os.environ["CUSTOM_TRAINED_MODEL_ID"]
+
+    async def get_manual_validation_info_from_recognize_custom_forms(self):
+        # the sample forms are located in this file's parent's parent's files.
+        path_to_sample_forms = Path(__file__).parent.parent.absolute() / Path("sample_forms/forms/Form_1.jpg")
+        from azure.core.credentials import AzureKeyCredential
+        from azure.ai.formrecognizer.aio import FormRecognizerClient
+        async with FormRecognizerClient(
+            endpoint=self.endpoint, credential=AzureKeyCredential(self.key)
+        ) as form_recognizer_client:
+
+            # The form you are recognizing must be of the same type as the forms the custom model was trained on
+            with open(path_to_sample_forms, "rb") as f:
+                forms = await form_recognizer_client.recognize_custom_forms(
+                    model_id=self.model_id, stream=f.read(), include_text_content=True
+                )
+
+            for idx, form in enumerate(forms):
+                print("--------RECOGNIZING FORM #{}--------".format(idx))
+                print("Form has type {}".format(form.form_type))
+                for label, field in form.fields.items():
+                    # each field is of type FormField
+                    # The value of the field can also be a FormField, or a list of FormFields
+                    # In our sample, it is not.
+                    print("Field '{}' has value '{}' based on '{}' within bounding box '{}', with a confidence score of {}".format(
+                        label,
+                        field.value,
+                        field.value_data.text,
+                        ", ".join(["[{}, {}]".format(p.x, p.y) for p in field.value_data.bounding_box]) if field.value_data.bounding_box else "N/A",
+                        field.confidence
+                    ))
+                for page in form.pages:
+                    print("-------Recognizing Page #{} of Form #{}-------".format(page.page_number, idx))
+                    print("Has width '{}' and height '{}' measure with unit: {}, and has text angle '{}'".format(
+                        page.width, page.height, page.unit, page.text_angle
+                    ))
+                    for table in page.tables:
+                        for cell in table.cells:
+                            print("Cell[{}][{}] has text '{}' with confidence {} based on the following words: ".format(
+                                cell.row_index, cell.column_index, cell.text, cell.confidence
+                            ))
+                            # text_content only exists if you set include_text_content to True in your function call to recognize_custom_forms
+                            # It is also a list of FormWords and FormLines, but in this example, we only deal with FormWords
+                            for word in cell.text_content:
```

Comment:
I thought in the api design, text content can be either a word or a line.  Lines don't have confidence.  @annelo-msft

Reply:
Yeah they can be either. I included a comment right above this code saying that in this sample we were only looking at FormWords, which have confidence. Let me know if I should make this clearer

Reply:
It might be better to iterate over a "content" (e.g. `for content in cell.text_content:`) and then check the type of the content within the loop.  We don't yet know of an example where the service returns a line text content, but it would be good for our sample to support this for when we run across one.